### PR TITLE
Fix/aag 2076 asset file requests not published correctly

### PR DESCRIPTION
**Pull request by thomas-james-1986**
This is a first pass at fixing the broken documentation page. 
Some internal navigation (sidebar) still doesn't appear to be fixed locally but it may just be a local issue

Screenshot 2022-05-24 at 21 25 41

Screenshot 2022-05-24 at 21 44 29

### CODE REVIEW

File: docs/Gemfile [hunk 1]
```diff
@@ -9,14 +9,14 @@ ruby RUBY_VERSION
...
 # Happy Jekylling!
-# gem "jekyll", "3.2.1"
+gem "jekyll", "4.2.2"
 
 # This is the default theme for new Jekyll sites. You may change this to anything you like.
 # gem "minima"
 
 # If you want to use GitHub Pages, remove the "gem "jekyll"" above and
 # uncomment the line below. To upgrade, run `bundle update github-pages`.
-gem "github-pages", group: :jekyll_plugins
```
Comment (tashmulla) [inline docs/Gemfile:19]: This was just for local development right?

File: docs/_includes/scripts.html [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
 <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
-<script src="{{ site.baseurl }}/assets/js/main.js"></script>
+<script src="{{ '/assets/js/' | relative_url }}main.js"></script>
```
Comment (tashmulla) [inline docs/_includes/scripts.html:3]: Ooh interesting, how come this one is `/assets/js/` as opposed to just `/assets/` similar to the rest of the changes? (Same question for the change in `docs/pages/search.html` 🙂

**Comment by tashmulla** [comment]
Nice one! To get the rest of the pages working, do you think it's a case of swapping out the use of `base_url` elsewhere? Might be good we do that anyway to keep consistency, but I can assume that could be an issue with the navigation?

File: docs/_includes/scripts.html (hunk 2 above)
Comment (thomas-james-1986) [reply docs/_includes/scripts.html:3]: Ah this was just due to the path where the file lives, most of them were in `assets` but some were in subfolders

**Comment by thomas-james-1986** [comment]
@tashmulla yes I think so, I'll remove `base_url` everywhere to wrap this up

File: docs/_includes/scripts.html (hunk 2 above)
Comment (tashmulla) [reply docs/_includes/scripts.html:3]: Although e.g. in `head.html`, the files lived in either the `/favicons` or `/css` subfolders as well right?

Comment (thomas-james-1986) [reply docs/_includes/scripts.html:3]: hmm yeah true, I think I need to match the folder structure everywhere?

Comment (tashmulla) [reply docs/_includes/scripts.html:3]: I don't think it matters too much, as long as `/assets` is there as the first folder to look into - just probably makes sense to have it consistent everywhere, so would just go with whichever way makes more sense for you 🙂

**Comment by gnhn** [comment]
If `baseUrl` is broken then how does it work for our other repositories?

**Pull request closed without merging**